guides/cpu-offloading/README.md

```diff
@@ -0,0 +1,136 @@
+# Well-lit Path: CPU Offloading
+
+## Overview
+
+This guide demonstrates how to deploy meta-llama/Llama-3.3-70B-Instruct using vLLM with CPU Offloading for KVCache. This guide has been validated on a cluster with 12xH100 GPUs split across three nodes.
+
+In this example, we will demonstrate a deployment of `Llama-3.3-70B-Instruct` with three vLLM instances
+
+## Hardware Requirements
+
+This guide requires 12 Nvidia H100 GPUs. It requires 2400 Gi of memory across all 3 pods (800 Gi memory for each pod).
+
+## Prerequisites
+
+- Have the [proper client tools installed on your local system](../prereq/client-setup/README.md) to use this guide.
+- Ensure your cluster infrastructure is sufficient to [deploy high scale inference](../prereq/infrastructure/README.md)
+  - You must have high speed inter-accelerator networking
+  - The pods leveraging inter-node EP must be deployed within the same networking domain
+- Configure and deploy your [Gateway control plane](../prereq/gateway-provider/README.md).
+- [Create the `llm-d-hf-token` secret in your target namespace with the key `HF_TOKEN` matching a valid HuggingFace token](../prereq/client-setup/README.md#huggingface-token) to pull models.
+- Have the [Monitoring stack](../../docs/monitoring/README.md) installed on your system.
+
+
+## Installation
+
+Use the helmfile to compose and install the stack. The Namespace in which the stack will be deployed will be derived from the `${NAMESPACE}` environment variable. If you have not set this, it will default to `llm-d-cpu-offloading` in this example.
+
+```bash
+export NAMESPACE=llm-d-cpu-offloading # or any other namespace
+cd guides/cpu-offloading/
+kubectl create namespace ${NAMESPACE}
+```
+
+### Create HuggingFace token
```

Comment:
This is already documented in the common prerequisites.

Reply:
I added this step because the prerequisite order was unclear; specifically, the namespace must exist before creating the Hugging Face token.

Reply:
let's just fix the prerequisites if it's unclear. This is pretty much used in every guide.